client/src/accounts/authentication/login.hook.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { describe, expect, it } from 'vitest';
+import { HTTP_BAD_REQUEST, HTTP_OK } from '~/api/api.constants';
+import { rest, server } from '~/mocks/server';
+import { act, renderHook, waitFor } from '~/test/utils';
+import { LoginForm } from './login-form.component';
+import { useLogin } from './login.hook';
+const ENDPOINT = '*/api/accounts/';
```

Comment:
Don't you have a `./vscode/settings.json`, I expected eslint/prettier to complain about this, we have rules on the ordering of imports and to use whitespace to categorize imports. Also, the const right below it, there should definitely be whitespace to separate imports from other logical code blocks.